lesson_05/zionbuchanan/lesson05.md

```diff
@@ -0,0 +1,8 @@
+## User Story
+### ZionBuchanan
+
+- As a Costco member, I want to add and customize food items selected from the food court menu, and recieve notifications with a timer showing how much time remains until my order is ready. 
```

Comment:
Don't forget that you need to speak to the purpose of the user story. Re-read the article cited in the homework details.